test/source/tests/setup.ts

```diff
@@ -41,6 +41,13 @@ export const defineSetupTests = (testVariant: TestVariant, testWithBrowser: Test
       await settingsPage.notPresent('.settings-banner');
     }));
 
+    ava.default('setup - optional checkbox for each email aliases', testWithBrowser(undefined, async (t, browser) => {
+      const settingsPage = await BrowserRecipe.openSettingsLoginApprove(t, browser, 'flowcrypt.compatibility@gmail.com');
+      await Util.sleep(5);
+      await SetupPageRecipe.createKey(settingsPage, 'unused', 'none', { key: { passphrase: 'long enough to suit requirements' }, usedPgpBefore: false, skipForPassphrase: true });
```

Comment:
> Actually, should we create a key with multiple emails (uids) when a user account has aliases, @tomholub?
> I found this line in setup-create-key.ts:

That would be for separate issue, of course, since it concerns generating new keys.
But I think maybe not. We just got scolded by some user who accused us of "commingling their identities together" that they wanted to keep separate. In general, people may be sensitive if we did this automatically, and it's too much of a burden to ask new users, especially if most of them will never understand the difference.

I think the compromise there would be to continue only using primary user email, but in the future add a way to modify / add identities in the settings, for users who care.

I'll file a separate issue, but not for anytime soon.

Reply:
Ok, just to let you know -- @martgil already added a checkbox for this on the key creation screen. Should it be removed?

Reply:
I see. We could possibly keep it, @martgil could you share screenshots of all the visual changes?

Reply:
Hello @tomholub  and @rrrooommmaaa ,

Sorry for the late response. Here it what it looks like: 

![image](https://user-images.githubusercontent.com/46025304/131247141-8f7f4a22-cf2c-4629-9889-814d754bbf12.png)

But the problem is that I have put the test on the "new encryption" phase, from which it should be on the import private key. I'm sorry to miss this kind of thing.

Reply:
The checkboxes are asking the user for which email addresses to submit the public key.

But Roman's question was about which identities should be encoded in the public key. These two are related, but not exactly the same.

To be honest, I wasn't planning on addressing this right now, as it requires proper thought and design before we implement it.

Besides, the issue was about importing public keys, not creating new ones.

I'd revert the changes on the key generation screen, and we can discuss it some other time in the future.